### ISSUE TITLE
Trace: Warning: entity.objectType is deprecated. Use entity.displayName instead

### ISSUE THREAD

**Issue by Listlessness** [PrismarineJS/mineflayer#3170]
When I make it, Trace: Warning: entity.objectType is deprecated. Use entity.displayName instead. Not sure what it means. Spams the console

**Comment by Listlessness** [comment]
Trace: Warning: entity.objectType is deprecated. Use entity.displayName instead
    at printObjectTypeWarning (/home/runner/SlimSeagreenMacro/node_modules/prismarine-entity/index.js:74:11)
    at set objectType [as objectType] (/home/runner/SlimSeagreenMacro/node_modules/prismarine-entity/index.js:43:7)
    at setEntityData (/home/runner/SlimSeagreenMacro/node_modules/mineflayer/lib/plugins/entities.js:180:25)
    at Client.<anonymous> (/home/runner/SlimSeagreenMacro/node_modules/mineflayer/lib/plugins/entities.js:204:5)
    at Client.emit (node:events:525:35)
    at Client.emit (node:domain:489:12)
    at emitPacket (/home/runner/SlimSeagreenMacro/node_modules/minecraft-protocol/src/client.js:83:12)
    at FullPacketParser.<anonymous> (/home/runner/SlimSeagreenMacro/node_modules/minecraft-protocol/src/client.js:107:9)
    at FullPacketParser.emit (node:events:513:28)

**Comment by That0neProdigy** [comment]
It would help if you sent the .JS file or the code you believe to be causing the issue

**Comment by lkwilson** [comment]
Bump to mineflayer version 1.13.0 or bump mineflayer entity back down to 2.2.0

**Comment by JohnKearney1** [comment]
Also happening with me, seems to spam the log anytime an entity is called.

**Comment by rom1504** [comment]
Please update mineflayer

**Comment by BlazingInferno1221** [comment]
So I updated mineflayer (even though I was supposed to already have the latest version since I installed it yesterday), and it stopped spamming this warning. I think what happened was that when I was installing, I gave into the
```
5 moderate severity vulnerabilities

To address all issues, run:
    npm audit fix
```
that popped up at the end of it. I can tell because when I ran "npm audit fix," that number of moderate severity vulnerabilities went down to 4, and the package number moved from 100 to 101... and because when I ran "npm update mineflayer" just now, the moderate severity vulnerabilities number went back to 5, and the package number dropped back to 100.
